CITATION.cff

```diff
@@ -0,0 +1,181 @@
+    description: Zenodo entry
+repository-code: 'https://github.com/iiasa/message_ix'
+url: 'https://docs.messageix.org/en/stable/index.html'
+abstract: >-
+  MESSAGEix is a versatile, dynamic, model framework for
+  energy-engineering-economy-environment (E4) systems
+  research.
+
+
+  MESSAGE (without “…ix”) is a specific formulation of a
+  generic linear programming (LP) optimization model for
+  strategic energy planning and integrated assessment of E4
+  systems, developed by the IIASA Energy, Climate, and
+  Environment (ECE) Program since the 1980s. To incorporate
+  feedback between prices and demand levels for energy and
+  commodities, the LP model can optionally be linked to the
+  economic general equilibrium (GE) MACRO model.
+
+
+  The message_ix Python package—also fully usable from
+  R—includes:
+
+      Implementations of MESSAGE, MACRO, and their linkage, in GAMS,
+      Application programming interfaces (APIs) and tools for model building and scientific programming,
+      Extensive documentation and a complete test suite.
+
+  The framework is built on IIASA's ix modeling platform
+  (ixmp), which provides data warehouse features for
+  high-powered numerical scenario analysis.
+keywords:
+  - integrated assessment
+  - energy systems
+  - macro-energy
+  - modelling
+  - scenario analysis
+  - data visualisation
+  - Python package
+license: Apache-2.0
+version: 3.6.0
+date-released: '2022-08-18'
+references:
+  - authors:
+    - given-names: Daniel
+      family-names: Huppmann
+      affiliation: IIASA
+      orcid: 'https://orcid.org/0000-0002-7729-7389'
+    - given-names: Matthew J.
+      family-names: Gidden
+      affiliation: IIASA
+      orcid: 'https://orcid.org/0000-0003-0687-414X'
+    - given-names: Oliver
+      family-names: Fricko
+      affiliation: IIASA
+      orcid: 'https://orcid.org/0000-0002-6835-9883'
+    - given-names: Peter
+      family-names: Kolp
+      affiliation: IIASA
+      orcid: 'https://orcid.org/0000-0003-0122-2839'
+    - given-names: Clara
+      family-names: Orthofer
+      orcid: 'https://orcid.org/0000-0002-7049-7791'
+    - given-names: Michael
+      family-names: Pimmer
+    - given-names: Nikolay
+      family-names: Kushin
+    - given-names: Adriano
+      family-names: Vinca
+      affiliation: IIASA
+      orcid: 'https://orcid.org/0000-0002-3051-178X'
+    - given-names: Alessio
+      family-names: Mastrucci
+      affiliation: IIASA
+      orcid: 'https://orcid.org/0000-0002-5611-7780'
+    - given-names: Keywan
+      family-names: Riahi
+      affiliation: IIASA
+      orcid: 'https://orcid.org/0000-0001-7193-3498'
+    - given-names: Volker
+      family-names: Krey
+      affiliation: IIASA
+      orcid: 'https://orcid.org/0000-0003-0307-3515'
+    title: "The MESSAGEix Integrated Assessment Model and the ix modeling platform (ixmp): An open framework for integrated and cross-cutting analysis of energy, climate, the environment, and sustainable development"
```

Comment:
I guess that the order doesn't really matter for the off format - so I'd first have the title, journal, year, then the authors, then the other attributes...